### DISCUSSION TITLE
Provider doesnt wait for guest OS to be running

### DISCUSSION THREAD

**Discussion by Vetrenik** [bpg/terraform-provider-proxmox#1549] [Q&A]
Hello everyone!

Using provider version 0.64 

When changing any resources of a vm, provider restarts this vm, but doesn't wait for guest OS to be running and routable

This leads to all vms, created with `for_each` to restart in non-HA manner, which breaks all ha apps, that are running on them 

Am i doing smth wrong?

**Comment by bpg-dev** [comment]
Hey @Vetrenik 👋🏼 

There is not much details in there to help you with your case and answer your question. 

Could you perhaps open a ticket, providing details about your config, network setup, output of `terraform apply` or debug logs? This would allow us to better understand your use case and provide some guidance. There is vast number of factors that may affect the behaviour you've described, so specifics does matter.

Thanks!

**Comment by Vetrenik** [comment]
Hi @bpg 

Thx a lot for a quick reply! 
I would perapare the sample in near future, but, probabply, after the start of a new quarter
